schema.sql

```diff
@@ -1,54 +1,74 @@
 /* Database schema to keep the structure of entire database. */
```

Comment:
- [ ] Kindly add the insert statements used in testing performance inside the `data.sql` as per this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_performance_audit.md#:~:text=If%20you%20decide%20to%20commit%20any%20INSERT%20INTO%20queries%20%2D%20remember%20to%20add%20the%20to%20the%20data.sql%20file).